.github/workflows/integration-tests-pr.yaml

```diff
@@ -0,0 +1,96 @@
+name: Integration Tests (Pull Request)
```

Comment:
I cannot really judge what/if anything is new in here..

Reply:
I just copied the old one